pages/portfolio.html

```diff
@@ -2,11 +2,58 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../css/protfolioStyle.css"/>
+  <title>Yanxian Zhang</title>
 </head>
 <body>
-  
+  <div class="headMessage">
+    <nav class="menu">
+      <ul>
+        <a href="index.html" target="_top"> Home </a>
+        <a href="about.html" target="_top"> About </a>
+        <a href="portfolio.html" target="_top"> PROTFOLIO </a>
+      </ul>
```

Comment:
When you have an unordered list (`ul` element), there must be list item (`li`) elements underneath before you can add things like links. So it should look like this:
```
<ul>
    <li><a href=......></li>
    <li><a href=......></li>
</ul>
```